src/main/java/org/mockito/InjectUnsafe.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito;
+
+import java.lang.annotation.*;
+
+/**
+ * Change the behavior of {@link InjectMocks}: allow injection of static/final fields that are normally skipped.
+ *
+ * <p>Typically, injection into instances is done via the constructor.</p>
+ *
+ * <p>In most cases you should refactor your code to support these best practices!</p>
+ *
+ * <p>However, sometimes you don't want to/cannot expose fields via constructor as this might change your api
+ * (even if the constructor is package-private).<br/>
+ * Even worse with static fields: you'd have to make setters available via method call.<br/>
+ * And then there's always legacy/library code that cannot be changed easily :(</p>
```

Comment:
Please add another warning about static, e.g. changing them might have repercussion on other part of the code because static fields can be used anywhere.